search-service/src/main/kotlin/com/egm/stellio/search/service/QueryService.kt

```diff
@@ -89,8 +90,8 @@ class QueryService(
                 .toList()
 
         val count = temporalEntityAttributeService.getCountForEntities(
-            temporalEntitiesQuery.ids,
-            temporalEntitiesQuery.types,
+            temporalEntitiesQuery.queryParams.id?.toSet() ?: emptySet(),
```

Comment:
strange move, you don't have an `ids` attribute in `QueryParams`?

Reply:
yes, rename it

Reply:
(and it could a `Set` instead of a `List` since there is no point in having duplicates and it avoids transformations into sets)